build/debian/changelog

```diff
@@ -1,277 +0,0 @@
-mixxx (2.2.2-0ubuntu1) bionic; urgency=medium
```

Comment:
We need most of the diebisn files to create the deb packages.
I know that there is some automation with cmake and pebuilder, but some files are required anyway.

To keep almost empty changelog is stupid anyway. I think we need to find a way to make our main changelog file debian compliment.

Reply:
Typical entry in the (RPMFusion) distribution changelog: "New upstream release". Otherwise only technical changes when bumping the .spec version.

This is a different kind of changelog.

Reply:
Not everyone uses a Debian system and the main changelog should stay agnostic.

Reply:
You can already build a DEB package via `cpack -G DEB`.